### DISCUSSION TITLE
Structure - Allow multiple menus

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3087] [Feature Requests]
I'd like to be able to manage more than one Menu structure (eg. Main menu, Footer menu, etc).

I would think that using the interface / functionality as per the Category Manager in EE, would be a nice way to achieve this.

**Comment by blue-dreamer** [comment]
What might help here is a new parameter for `{exp:structure:nav}` to filter output based on a custom field, much the same as we can use the search parameter in regular channel entry tags. That way you could use the custom field to filter nav entries according to your needs. If you have two navs the custom field could have the settings you want, e.g. "none", "header", "footer", "both" for authors to specify which nav it should be added to.

Examples:
```

{exp:structure:nav search:my_custom_field="header"}
Only show entries where my_custom_field == "header"
{/exp:structure:nav}

{exp:structure:nav search:my_custom_field="footer"}
Only show entries where my_custom_field == "footer"
{/exp:structure:nav}

{exp:structure:nav search:my_custom_field="header|footer"}
Only show entries with either "header" or "footer" selected.
{/exp:structure:nav}
```
This would open up a few possibilities, filter navs by author, custom field, day/date, past/future entries, status... think what you could do with it.
